sbin/common/config_init.sh

```diff
@@ -366,6 +367,35 @@ function configDefaults() {
   # The OS kernel name, e.g. 'darwin' for Mac OS X
   BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
 
+  # Determine OS full system version
+  local unameSys=$(uname -s)
+  local unameOSSysVer=$(uname -sr)
+  if [ "${unameSys}" == "Linux" ]; then
+    # Linux distribs add more useful distrib /etc/*-release file content
+    set +e
+    cat /etc/*-release > /dev/null
+    local cmdRc="$?"
+    if [ "${cmdRc}" == "0" ]; then
+      local linuxName=""
+      cat /etc/*-release | grep PRETTY_NAME > /dev/null
+      cmdRc="$?"
+      if [ "${cmdRc}" == "0" ]; then
+        linuxName=$(cat /etc/*-release | grep PRETTY_NAME | cut -d'=' -f2 | tr -d '"')
```

Comment:
Maybe:
```
linuxName=$(grep PRETTY_NAME /etc/*-release | cut -d= -f2 | tr -d '"')
```
Or even this, but it's a bit less clear what's happening so the grep might be better:
```
linuxName=$(awk -F= "/PRETTY_NAME/{print$2}" | tr -d '"')
```

Reply:
prefer grep